api/location/models.py

```diff
@@ -1,9 +1,9 @@
-from sqlalchemy import (Column, String, Integer, Enum, Index)
+from sqlalchemy import (Column, String, Integer, Enum, Index, ForeignKey)
 from sqlalchemy.orm import relationship
 from sqlalchemy.schema import Sequence
 
 from helpers.database import Base
-
+from api.office_structure.models import OfficeStructure  # noqa: F401
 from utilities.utility import Utility, StateType, cascade_soft_delete
```

Comment:
I believe you can remove this `noqa` here.

Reply:
@kampkelly This flake8 issue is actually quite confusing. Not importing the `OfficeStructure` will result in an undefined error for line 37 while importing result in `unused import` violation, hence the use of `noqa: F401`.

Reply:
You are not using this import, that is why it is saying 'imported but unused' when you remove the noqa.
You can remove the import because line 37 is `structure = relationship('OfficeStructure')`.

Reply:
I have removed the import and the unexpected error is no more. This is a strange behavior though.